src/lib/Ska/Parse_CM_File.pm

```diff
@@ -699,6 +699,11 @@ sub MM {
         $manvr_hash{roll}          = $1 if ($final_attitude =~ /ROLL\s*\(deg\):\s+(\S+)/);
         $manvr_hash{dur}           = $1 if ($output_data =~ /Duration\s*\(sec\):\s+(\S+)/);
         $manvr_hash{angle}         = $1 if ($output_data =~ /Maneuver Angle\s*\(deg\):\s+(\S+)/);
+        my @init_quat              = ($1,$2,$3,$4) if ($initial_attitude =~ /Quaternion:\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)/);
```

Comment:
Is this too much new Perl?  Seemed to kind of make sense for this.  I'll look at parse_cm next.